### ISSUE TITLE
Add custom "Multi Selector" switches - Feature request

### ISSUE THREAD

**Issue by ksga** [mycontroller-org/mycontroller#340]
Add the possibility to edit predefined multi state switches, or possible a "Dummy" device with custom states.
Something like what is possible in [Domoticz ](https://www.domoticz.com/forum/viewtopic.php?t=7244#p66136)would be nice.

**Comment by fab33** [comment]
But please don't implement this like in domoticz. Prefer OpenHab Way (selector) and the ability to dispay selector as multiple button with one associated value for each element.

**Comment by jkandasa** [comment]
@fab33 You mean to create it in one place and use it for many switches. When you want to change, change it in one place? Am I right?

can you give OpenHab screen shot or kind of something?

**Comment by fab33** [comment]
No, it a multi-state switch. For example in France we have a system for electric heating devices with 6 different order (Halt, Freezing, Eco-2, Eco-1, Eco, Confort). You can see it in Domoticz example @ksga link
You can see this example : https://github.com/openhab/openhab/wiki/Nest-Binding-Example with HVAC Mode
And if you prefer in Openhab you just have to change Sitemap to have selector.
Another cool feature (I don't know if OH support it) is to make group to control many system. However, you can do it with scene. This group feature can be fine with percentage (like roller shutters).
